Containers/borgbackup/Dockerfile

```
@@ -1,14 +1,13 @@
-FROM debian:bullseye-20221219-slim
+FROM alpine:3.17.0
```

```suggestion
FROM alpine:3.17.1
```
3.17.1 is now available on docker hub

---

done